client/lib/signup/test/step-actions.js

```diff
@@ -89,6 +95,151 @@ describe( 'createSiteWithCart()', () => {
 	} );
 } );
 
+describe( 'isDomainFulfilled', () => {
+	const submitSignupStep = jest.fn();
+
+	beforeEach( () => {
+		flows.excludeStep.mockClear();
+		submitSignupStep.mockClear();
+	} );
+
+	test( 'should remove a step when there is greater than one domain', () => {
```

Comment:
I'd be tempted to write two separate test blocks for this, as we're really testing two things -
"It calls `submitSignupStep` with X params" and "It calls `flows.excludeStep` with stepName" - the actual removing of the step is a detail of those called functions which are separate testable 'units'.

What do you think? This totally isn't a blocking comment - just a hunch from me!

Reply:
Good call! I've split this one into two separate tests. I was a little torn with some of these, and the descriptions of the tests, between giving meaningful descriptions of the desired outcome, and testing just the one behaviour. But what you're saying here makes sense to me.

I've left some of the other similar tests as-is, but happy to go back over them if you think they should be split up, too!